test/unitTests/ignoreFiles/ignoreFiles.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, test } from '@jest/globals';
+import { writeFileSync } from 'fs';
+import { join } from 'path';
+import { pathToFileURL } from 'url';
+import { readFileSyncNorm } from '../../../lib/cli/readFileSyncNorm';
+import { ProjectParser } from '../../../lib/projectParser';
+import { VhdlLinter } from '../../../lib/vhdlLinter';
+
+async function getProjectParser(ignoreGlob: string) {
+  const setting = `{paths: {ignoreFiles: ["${ignoreGlob}"]}}`;
+  writeFileSync(join(__dirname, "vhdl-linter.yml"), setting);
+  return await ProjectParser.create([pathToFileURL(__dirname)]);
+}
+
+test.each([
+  ["**/*entity*", true], // by filename
+  ["**/*no_match*", false], // by filename
+  ["**/export/**/*", true], // by last folder
+  ["**/modules/**/*", true], // by middle folder
+  ["src/*", false], // by first folder
+  ["src/**/*", true], // by first folder
```

Comment:
Yeah again , i do not like this behavior. I do not think this is clear from user perspective.
The first think ignores the src/modules folder I think this should then everythink inside.